GithubProfileWiki/Screens/FollowersList/FollowersListViewModel.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  FollowersListViewModel.swift
+//  GithubProfileWiki
+//
+//  Created by ilter on 23.01.2022.
+//
+
+import Foundation
+
+class FollowersListViewModel {
+    func fetchFollowers(userName: String, param: [String: Any], completion: @escaping (Followers?, Error?) -> ()) {
+        let request = FollowersAPI(userName: userName)
+        
+        let apiService = APIService(apiRequest: request)
```

Comment:
if you can separate request parameter from init of APIService, you dont have to create a new insance of it for each network call

Reply:
I could not understand very well, can you explain more :(